generate_upstream_list.py

```diff
@@ -9,27 +9,23 @@
 
 def get_commit_log(git_repo, viewvc_url, author_in_commit_body=False):
     repo = Repo(git_repo)
-    log = repo.commits(config.EMAIL_GREP, search_body=author_in_commit_body)
+    email_grep = '(Contributed|Patch) (from|by).*@opera.com'
```

Comment:
It's no biggie.  The ones using it (that we know of) are single people using it to keep their own list of Chromium work.  Though the intention was never for this to be some sort of tracker.  Though it was easy to separate the things out so I did it to be nice.  Separating out the domain would make it simpler for someone else to check their upstreams (like Brave) or even to later-on have something that checks even more.

But I think it's quite fine as-is.  Separating out domain will make it better.  I mostly wanted to tell you the reasoning of why it was in config before. :)